### DISCUSSION TITLE
Add the AdvancedSlimePaper patches

### DISCUSSION THREAD

**Discussion by 0xIO32** [PurpurMC/Purpur#1366] [Feature Requests]
It would be great if these patches from this project would be included: https://github.com/InfernalSuite/AdvancedSlimePaper/
It adds the capability for the AdvancedSlimePaper plugin to hook into the world loading.
It is able to load worlds over other world formats like mysql, mongodb oder a file with the "slime" format, which will be loaded in the memory.
The Result is that the worlds are faster loading and this is extremely good for example bedwars maps, which are not that big.

The Plugin was originally developed by some of the Hypixel Devs and is now maintained by some of the Paper devs and other contributors.

There is also an fork which adds these patches, but is not always up to date: https://github.com/play-rivalsnetwork-hu/AdvancedSlimePurpur

**Comment by TheRealRyGuy** [comment]
This is not just pulling the patches, it's also pulling multiple swm modules (which iirc aren't even deployed? would require a sub module setup)
more work for maintainers, would be fun though

**Reply by 0xIO32** [reply]
Or we could just create patches from the swm modules.

**Comment by 0xIO32** [comment]
![image](https://github.com/PurpurMC/Purpur/assets/77546847/4d854b71-1b4f-42b2-8c44-6bb1bcbac8fb)
Official announcement from their discord.
